src/components/lib/musl-1.1.11/bin/musl-gcc

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-exec "${REALGCC:-gcc}" "$@" -specs "/home/ryx/research/composite/src/components/lib/musl-1.1.11/lib/musl-gcc.specs"
+exec "${REALGCC:-gcc}" "$@" -specs "/home/georgit/research/composite/src/components/lib/musl-1.1.11/lib/musl-gcc.specs"
```

Comment:
You should probably talk to Yuxin about how to make this generic.  This needs to be configured in some way (e.g. see `Makefile.cosconfig` for an example).  If you and Yuxin come up with something, let me know.